### PR TITLE
README.md updates

### PR DESCRIPTION
**Pull request by zackorenberg**
Added descriptions of some core widgets to README.md

### CODE REVIEW

File: README.md
Comment (Bachibouzouk) [inline README.md]:
```suggestion
Among these are several designed to help analysis/manipulation of data and aid in seemless communication with devices.
```

File: README.md
Comment (Bachibouzouk) [inline README.md:492]:
```suggestion
Built into LabGUI, there are a set of core widgets for basic operations.
```

Comment (Bachibouzouk) [reply README.md:492]: You can add all commits as one if you choose "Add suggestion to batch"

File: README.md
Comment (Bachibouzouk) [inline README.md]:
```suggestion
These widgets can be displayed/hidden from the `Window` dropdown menu.
```

File: README.md
Comment (Bachibouzouk) [inline README.md]:
```suggestion
This widget allows to the user to specify where data is saved, as well as leave some notes as comments in that datafile. Notes left by the user are saved/loaded by default.
```

File: README.md
Comment (Bachibouzouk) [inline README.md]:
```suggestion
This is where users defines their experiment (a sweep, timeseries, etc...)
```

File: README.md
Comment (Bachibouzouk) [inline README.md]: I initially designed it so that one can provide variables in the script. And change their values live

File: README.md
Comment (Bachibouzouk) [inline README.md]:
```suggestion
This widget intercepts all strings sent to the `Standard Output` and prints them within a TextEdit, acting as a read-only terminal window.
```

**Review by Bachibouzouk — APPROVED** [review]
Thanks for updating the readme!!!

File: README.md
Comment (zackorenberg) [reply README.md:492]: That selection is not enabled for me. My only issue with this one is that the core widgets are not designed for basic operation, but rather it forms a basis for all operations. Thoughts on how to reword in order to reflect that?

File: README.md
Comment (zackorenberg) [reply README.md]: This widget is oriented more for readability of data files. As such, it will save certain 'variables' with values, which can be manually changed by the user, for managing datafiles afterwards. This may be especially useful if the user is planning on pragmatically analyzing their data independently, as the variables are set in the header in a way which can be easily read by using the LabGUI library (specifically the function IOTool.load_file_windows, where it is retrievable as a tuple under the second return variable, label. If someone analyzing their data were to user IOTool.load_file_windows to load multiple datafiles, it could then be easily organized by certain assigned user data value(s).

Comment (zackorenberg) [reply README.md]: Please refer to the latest commit for added documentation on this widget

File: README.md
Comment (zackorenberg) [reply README.md]: Corrected manually

**Comment by zackorenberg** [comment]
@Bachibouzouk awaiting your approval for my most recent commit on this branch before merging.

**Comment by Bachibouzouk** [comment]
@zackorenberg - I had approved this branch

**Review by Bachibouzouk — APPROVED** [review]

**Pull request merged**
